polyply/data/2016H66/polyether_blocks.ff

```diff
@@ -13,3 +13,15 @@ PEO 3
 3 2 2 gb_18 {"comment":"CHn-OA,OE"}
 [ angles ]
 3 2 1 2 ga_15 {"comment":"CHn-CHn-C,CHn,OA,OE,NR,NT,NL"}
+
+[ moleculetype ]
+mPEG 3
```

Comment:
so, does this just define PEO that can be combined with PMMA? Maybe a one-line comment to say this in short? Because I see that the block itself is equivalent to PEO.
Not sure we want to go this way (i.e, adding some more comments on the `.ff` file) but it could be useful.

Reply:
pretty much! Using this procedure you get away with "just" making a branched graph while otherwise you'd also have to add an edge label.